src/avatar/index.tsx

```diff
@@ -2,36 +2,37 @@ import React from 'react'
 import cc from 'classcat'
 
 import style from 'avatar/style'
-import Check from 'icon/checkIcon'
 import prefix from '_utils'
-import { color } from '_utils/branding'
 
-export interface IdCheck {
-  readonly checked?: boolean
-  readonly title?: string
-}
-
-export interface AvatarInterface extends IdCheck {
+export interface AvatarInterface {
   readonly className?: Classcat.Class
   readonly image?: string
   readonly alt?: string
   readonly medium?: boolean
   readonly large?: boolean
+  readonly bottomBadge?: string | JSX.Element
+  readonly topBadge?: string | JSX.Element
 }
 
-export const IdCheck = ({ checked = false, title = null }: IdCheck) => (
-  <span className="kirk-idCheck">
-    <Check size="100%" title={title} iconColor={color.white} />
-    <style jsx>{style}</style>
-  </span>
-)
-
-const Avatar = ({ medium, large, className, checked, image, alt = '', title }: AvatarInterface) => (
-  <div className={cc([prefix({ medium, large, image: !!image }), className])}>
-    {image && <img src={image} alt={alt} />}
-    {checked && <IdCheck checked={checked} title={title} />}
-    <style jsx>{style}</style>
-  </div>
-)
+const Avatar = ({
+  medium,
+  large,
+  className,
+  image,
+  alt = '',
+  bottomBadge,
+  topBadge,
+}: AvatarInterface) => (
+    <div
+      className={cc([
+        prefix({ medium, large, image: !!image }, 'kirk-avatar-'),
+        className, 'kirk-avatar',
+      ])}>
+      {image && <img src={image} alt={alt} />}
+      {topBadge}
```

Comment:
I didn't know we had two ways of putting badges on the avatar. 
Moroever, you have topBadge and bottomBade, but nothing is actually placing them right?

Reply:
Agreed, nothing places them automatically to the top or bottom.
As you suggested, I won't allow for any JSX.Element there but preset a `UnreadNotificationsBadge` and `IdCheckBadge` instead.